app/services/data/get-current-workload-report.js

```diff
@@ -0,0 +1,11 @@
+const config = require('../../../knexfile').app
```

Comment:
Slightly uncomfortable with the name of this file, as it's not really returning the current workload report. It's returning an array of open workload reports. Maybe get-open-workload-reports?

Reply:
renamed